examples/hub-values.yaml

```diff
@@ -41,6 +42,12 @@ pachd:
   metricsEndpoint: "https://metrics.test/api/v1/metrics"
   service:
     type: "ClusterIP"
+  externalService:
+    enabled: true
+    loadBalancerIP: "192.168.1.1"
```

Comment:
Ditto here: 192.168.0.0/16 is reserved for private use and 192.168.1.1 is not unlikely to be in use on any given network. Something from the 192.0.2.0/24 range would be safer.

I know: I really _do_ need to get out more!